src/components/colorscale/attributes.js

```diff
@@ -34,7 +34,13 @@ module.exports = {
     colorscale: {
         valType: 'colorscale',
         role: 'style',
-        description: 'Sets the colorscale.'
+        description: [
+            'Sets the colorscale.',
+            'The colorscale must be an array containing',
+            'arrays mapping a normalized value to an rgb color. At minimum, a',
```

Comment:
It doesn't have to be an rgb color. rgba, hex, hsl should be fine as described [here](https://github.com/plotly/plotly.js/blob/master/src/lib/coerce.js#L114-L122)